components/match2/wikis/halo/match_group_input_custom.lua

```diff
@@ -293,16 +295,23 @@ end
 function matchFunctions.getTournamentVars(match)
 	match.mode = Logic.emptyOr(match.mode, Variables.varDefault('tournament_mode', 'team'))
 	match.publishertier = Logic.emptyOr(match.publishertier, Variables.varDefault('tournament_publishertier'))
-
-	match = MatchGroupInput.getCommonTournamentVars(match)
-
-	match.game = _GAME[match.game or '']
-
-	return match
+	return MatchGroupInput.getCommonTournamentVars(match)
 end
 
 function matchFunctions.getVodStuff(match)
-	match.stream = Streams.processStreams(match)
```

Comment:
didn't this line do exactly what the lines below do now?
if so i personally would keep the old line